pages/c/_cluster/index.vue

```diff
@@ -1,5 +1,5 @@
 <script>
```

Comment:
This whole file needs to move to pages/c/_cluster/overview.vue and the nav link updated so that its route is not always active

Reply:
I dont think I actually do. I added an exact flag to the types map and related files to allow us to control this because I think having to create fake pages that redirect because of the link highlighting is not the right move. I didn't see any negative effects from adding this.